data/pages-content/cfp.md

```diff
@@ -8,7 +8,7 @@ subtitle: EuroPython Call for Proposals
 **EuroPython 2024 Call for Proposals (CFP) is Open.**
 
 It will be running between
-**Wednesday, 14th February 2024** and **Wednesday, 6th March 2024**
+**Wednesday, 14th February 2024** and **Wednesday, <s>6th</s> 13th, March 2024**
```

Comment:
I hope the tag `<s>` works.

Reply:
![Screenshot 2024-03-07 at 23-29-22 CFP July 8th-14th 2024 Prague Czech Republic   Remote](https://github.com/EuroPython/website/assets/177982/93b3aeae-84bc-4cac-a0dd-c9e28f3b15dc)

It does, but it's a bit unclear.